modules/ui/map3d_viewer.js

```diff
@@ -225,6 +226,7 @@ export function uiMap3dViewer(context) {
       if (d3_event) d3_event.preventDefault();
 
       _isHidden = !_isHidden;
+      if (context.systems.urlhash?.getParam('map3d')) _isHidden = true;
```

Comment:
The meaning of your code is backwards here, I think- this seems to be saying 'if we have a 3d map param, we should hide!'.

Reply:
Refactored the toggle function to make it clearer, thanks Ben!